doc/source/poppy_config.rst

```diff
@@ -0,0 +1,72 @@
+
+=========================
+Poppy Configuration Notes
+=========================
+
+Description
+-----------
+
+Please find below various poppy conf file configuration beside which in comments you will find the necessity of those configurations.
+These are some of the important configurations and also some regular configurations, not all the configuration information are explained below.
+
+Default configuration
+---------------------
+
+Make sure to keep the logging.conf in the correct directory
+
+[DEFAULT]
+verbose = True 			#To get verbosity for poppy server and worker
+debug = True 			#to get debug logs for poppy server and worker
+base_url = http://0.0.0.0:8888	#To have accessibility from anywhere
+
+log_config_append = /Users/<user>/.poppy/logging.conf   
```

Comment:
Might be too mac specific -- maybe instead we can say:

`${HOME}/.poppy/logging.conf`